scipy/special/tests/test_basic.py

```diff
@@ -178,9 +178,6 @@ def test_berp(self):
     def test_besselpoly(self):
         assert_equal(cephes.besselpoly(0,0,0),1.0)
 
-    def test_btdtria(self):
-        assert_equal(cephes.btdtria(1,1,1),5.0)
-
```

Comment:
I do not think that the expected value of 5 makes sense, so I deleted this test.